tegel/src/components/theme/colour/colour-scales.stories.js

```diff
@@ -0,0 +1,60 @@
+export default {
+  title: 'Foundations/Colour',
+  parameters: {
+    layout: 'fullscreen',
+    docs: {
+      source: {
+        code: 'closed',
```

Comment:
```suggestion
        state: 'closed',
```
This fix will have the source closed by default 🙂

Reply:
Nice!